static/docs/command-reference/config.md

```diff
@@ -79,6 +79,12 @@ This is the main section with the general config options:
 - `core.checksum_jobs` - number of threads for computing checksums. Accepts
   positive integers. The default value is `max(1, min(4, cpu_count() // 2))`.
 
+- `core.hardlink_lock` - use `hardlink`-based locks instead of `flock`-based
+  ones (e.g. repository lock `.dvc/lock`). Useful when you have your DVC
+  repository on a filesystem that doesn't properly support file locking (e.g.
+  [NFS v3 and older](http://nfs.sourceforge.net/)). Accepts values `true` and
+  `false` (default).
```

Comment:
Suggested text:

- `core.hardlink_lock` - use `hardlink` locks instead of the default ones, based on [`flock`](https://linux.die.net/man/2/flock) ones (i.e. project lock file `.dvc/lock`). Accepts values `true` and `false` (default). Useful when you have your DVC project on a file system that doesn't properly support file locking (e.g. [NFS v3 and older](http://nfs.sourceforge.net/)).

But also probably link "lock" to somewhere else where locking is explained, although that may be pending in a separate PR, so could do this over there also.

Reply:
p.s please read the above message in GH as I have edited it. Had some errors.

This message will self destruct in 5 4 3 2 1

Reply:
Adjusted. Created https://github.com/iterative/dvc.org/issues/876

Reply:
@efiop remove the previous block? we have two versions at the same moment now.

Reply:
Oops, fixed.